src/main/java/com/openlattice/shuttle/dates/JavaDateTimeHelper.java

```diff
@@ -59,18 +58,18 @@ public LocalDate parseDate(String date) {
     }
 
     public OffsetDateTime parseDateTime(String date) {
-        if (shouldIgnoreValue(date))
+
+        if ( shouldIgnoreValue(date) ) {
             return null;
+        }
+
         for (int i = 0; i < datePatterns.length; ++i) {
             DateTimeFormatter formatter = formatters.get(i);
             try {
                 LocalDateTime ldt = LocalDateTime.parse(date, formatter);
-                //                if (ldt.isBefore(LocalDateTime.of(1900, 1, 1, 0, 0, 0))){return null;}
-                if (
-                        datePatterns[0].matches(".*yy.*") && !datePatterns[0].matches(".*yyyy.*") ||
-                                datePatterns[0].matches(".*YY.*") && !datePatterns[0].matches(".*YYYY.*")
-                ) {
-                    if ((ldt.getYear() - LocalDate.now().getYear()) > 20) {
+                if ( datePatterns[0].matches(".*yy.*") && !datePatterns[0].matches(".*yyyy.*")
```

Comment:
does this need parenthesis around the AND pairs? (assuming this is supposed to be `(A && B) || (C && D)`)

Reply:
Huh. I think you found a bug since it's definitely not that way prior to this reformatting.

Reply:
I'll add some tests

Reply:
https://jira.openlattice.com/browse/LATTICE-1977